includes/class-admin-views.php

```
@@ -924,28 +999,31 @@ public function render_additional_fields( $form_id = 0, $context = 'single' ) {
 			}
 
 			// Render a label for each of them
-			echo new GravityView_Admin_View_Field( $item['label_text'], $item['field_id'], $item, $settings = array(), $form_id, $form );
-
+			echo new GravityView_Admin_View_Field( $item['label_text'],
+				$item['field_id'],
+				$item,
+				$settings = [],
```

PHPMD (2.15.0)

Avoid unused local variables such as '$settings'. (Unused Code Rules)

(UnusedLocalVariable)